test/SlowTests/Server/Documents/PeriodicBackup/PeriodicBackupSlowTests.cs

```diff
@@ -264,6 +264,63 @@ public class PeriodicBackupTestsSlow : RavenTestBase
         }
 
         [Fact(Skip = "RavenDB-7931 - Takes too long"), Trait("Category", "Smuggler")]
+        public async Task CanImportTombstonesFromIncrementalBackup()
```

Comment:
Please have a test just for this smuggler functionality, aside from periodic backup

Reply:
I tested it in periodic back because that we only export and import tombstones for incremental backups. And we don't have for now `store.Smuggler.ExportIncremenalAsync()`.

Reply:
okay, let create an issue to get to it before RTM

Reply:
We have this issue: http://issues.hibernatingrhinos.com/issue/RavenDB-7931
and I don't think we want to create ExportIncremenalAsync method. Are we?

Reply:
okay, that's enough